modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/build_fragment.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function buildFragment(htmlString) {
+	const tempDiv = document.createElement('div');
+	tempDiv.innerHTML = `<br>${htmlString}`;
+	tempDiv.removeChild(tempDiv.firstChild);
+
+	const fragment = document.createDocumentFragment();
+	while (tempDiv.firstChild) {
+		fragment.appendChild(tempDiv.firstChild);
+	}
+
+	return fragment;
+}
```

Comment:
If you can be bothered, as well as moving this in here you could apply a couple of style tweaks: some extra blank lines in the places Brian usually asks for them; and I'd avoid the `temp` abbreviation here (just call it `div`).

Reply:
I can be bothered, but just because you asked nicely. Forced it to https://github.com/liferay-frontend/liferay-portal/pull/530/files#diff-b126da00065bb5c3653eae3844e0acef49f3764d87a1b7b0cccbe85753f0fdfc

Going to hit forward and see if we get lucky the first time.